theseus_gui/src/pages/project/Index.vue

```diff
@@ -29,9 +29,14 @@
         </Categories>
         <hr class="card-divider" />
         <div class="button-group">
-          <Button color="primary" class="instance-button" @click="install(versions[0].id)">
+          <Button
```

Comment:
While a project is installing/the future is being awaited, this have a loading spinner and say "Installing..." or something similar (same with all other cases of an install button)

Reply:
Users also shouldn't be able to install a project twice- only "update" it/change the version